CsScriptManaged/CsScripts/DebugEventInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
```

Comment:
This file is targeting wrong folder. It looks like file is stale and should be deleted.